docsite/rst/playbooks_vault.rst

```diff
@@ -14,7 +14,7 @@ What Can Be Encrypted With Vault
 
 The vault feature can encrypt any structured data file used by Ansible.  This can include "group_vars/" or "host_vars/" inventory variables, variables loaded by "include_vars" or "vars_files", or variable files passed on the ansible-playbook command line with "-e @file.yml" or "-e @file.json".  Role variables and defaults are also included!
 
-Because Ansible tasks, handlers, and so on are also data, these can also be encrypted with vault.  If you'd like to not betray what variables you are even using, you can go as far to keep an individual task file entirely encrypted.  However, that might be a little much and could annoy your coworkers :)
+Because Ansible tasks, handlers, and so on are also data, these too can be encrypted with vault.  If you'd not like to betray even the variables you are using, you can go as far as keeping individual task files entirely encrypted.  However, that might be a little too much and could annoy your coworkers :)
```

Comment:
A couple suggestions on your suggestions:

* I find "If you'd not like to betray even the variables you are using" to be confusing compared to the original wording.  Maybe just removing "even" within the "you are even using" fragment would suffice.
* "a little much" is a common phrase, I don't see a need to change that.

Also, "patch 1" or "patch 2" as the title of the PR isn't very great, I'd change it to reflect that you're making grammar changes (as your commit messages state).